src/components/Reply/Reply.tsx

```diff
@@ -1,42 +1,63 @@
-import { Data } from "@/types/types";
+import { Comment, Data } from "@/types/types";
 import UserContext from "@/UserContext";
 import { SetStateAction, useContext, useState } from "react";
 import styles from "./Reply.module.css";
 
 interface ReplyProps {
+  parrentComment?: Comment;
```

Comment:
Мелочь, но заставляет страдать перфекционистов
```suggestion
  parentComment?: Comment;
```
Расширение [code spellchecker](https://marketplace.visualstudio.com/items/?itemName=streetsidesoftware.code-spell-checker) помогает